cg/meta/report/templates/macros/data_analysis/data_analysis.html

```diff
@@ -38,6 +38,7 @@ <h4 class="mt-4 mb-3">Dataanalys</h4>
         <th scope="col">Bioinformatisk analys</th>
         <th scope="col">Genomversion</th>
         <th scope="col">{% if "balsamic" in workflow %} Analystyp {% endif %}</th>
+        <th scope="col">{% if "balsamic" in workflow and case.data_analysis.pons != "N/A" %} Panel of Normals {% endif %}</th>
```

Comment:
@mathiasbio 

Is there a technical word for the Panel of Normals in Swedish? `Panel av Normala` sounds a bit weird to me 🤔

Reply:
And I've tentatively included it as part of the data analysis table. If there are multiple PONs, they will be shown separated by commas, along with the tools used, in the name.

Feel free to suggest other layouts. It can also be included in its own section (2 min work 😋 )

Reply:
I think Panel of Normals is generally known and accepted terminology...I don't think anyone who wouldn't understand what a Panel of Normals was in english would know what it was even if we translated it to some reasonable swedish word 😂 I think we can keep the english!

Reply:
I don't see a point at the moment to add a separate table for it. Looks like a good choice to me to put it into the Data-analysis table as it fits and I don't see any extra information that we might want to add 🤔 so I like it!

Reply:
Cool thanks @mathiasbio!